### PR TITLE
importer: tests

### PR DESCRIPTION
**Pull request by ryan-farley**
Added tests for Netscape and Mozilla formats.

---
This change is [Reviewable](https://reviewable.io/reviews/qutebrowser/qutebrowser/3293)

### CODE REVIEW

File: tests/unit/scripts/test_importer.py [hunk 1]
```diff
@@ -19,9 +19,33 @@
 # along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
 
 import pytest
+import os
```
Comment (The-Compiler) [inline tests/unit/scripts/test_importer.py]: Please separate imports into Python/third-party/qutebrowser blocks:

```python
import os

import pytest

from scripts import importer
```

File: tests/unit/scripts/test_importer.py [hunk 2]
```diff
@@ -19,9 +19,33 @@
...
 from scripts import importer
 
-_chrome_profile = 'tests/unit/scripts/chrome-profile'
+_samples = 'tests/unit/scripts/importer_sample'
+
+
+def qm_expected(input_format):
+    """Read expected quickmark-formatted output."""
+    with open(os.path.join(_samples, input_format, 'quickmarks'), 'r', encoding='utf-8') as f:
```
Comment (The-Compiler) [inline tests/unit/scripts/test_importer.py]: Please break lines if longer than 79 chars, see [pylint output](https://travis-ci.org/qutebrowser/qutebrowser/jobs/302260842#L487).

Comment (ryan-farley) [reply tests/unit/scripts/test_importer.py]: Sorry about that; for some reason it didn't show up on my local pylint run.

**Comment by The-Compiler** [comment]
Thank you!

**Pull request merged**